conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -470,11 +481,10 @@ func (algodImp *algodImporter) getBlockInner(rnd uint64) (data.BlockData, error)
 
 	nodeRound, err := waitForRoundWithTimeout(algodImp.ctx, algodImp.logger, algodImp.aclient, rnd, waitForRoundTimeout)
 	if err != nil {
-		// If context has expired.
 		if algodImp.ctx.Err() != nil {
-			return blk, fmt.Errorf("GetBlock ctx error: %w", err)
+			return blk, fmt.Errorf("importer algod.GetBlock() ctx cancelled: %w", err)
```

Comment:
let the context error say what the error is:
```suggestion
			return blk, fmt.Errorf("importer algod.GetBlock() ctx err: %w", algodImp.ctx.Err())
```

What do you think about removing the `algodImp.ctx.Err()` check completely? It doesn't seem to add a lot here and is equally applicable to all the other spots where an error is returned.

Reply:
Yeah, we could combine into something like:
```go
if err != nil {
    err = fmt.Errorf("waitForRoundWithTimeout err: %w, ctx cancellation: %w", err, algodImp.ctx.Err())
    algodImp.logger.Errorf("importer algod.GetBlock() called waitForRoundWithTimeout: %v", err)
    return data.BlockData{}, err
}
```

Reply:
I'll amend the PR.

Reply:
I don't thinnk `algodImp.ctx` has much of anything to do with the rest of this code. If so leaving it out completely should be fine

Reply:
Thinking more, it is kind of weird that we're reporting the cancellation error here, and not in the next section. And we can assume that the Pipeline is in control of the context and it should be in charge for reporting its own cancellation.

I'll amend again to remove the `algodImp.ctx.Err()` printout.